src/amy.h

```diff
@@ -98,7 +102,8 @@ enum coefs{
 #define AUDIBLE_SUSPENDED 4
 #define IS_MOD_SOURCE 5
 #define IS_ALGO_SOURCE 6
-#define STATUS_OFF 7
+#define TRANSFER_DATA 7
```

Comment:
We have weirdly mixed together osc[].status values with event.status.  I don't think there's any reason to mix them - I think event.status is only EMPTY, SCHEDULED, and now TRANSFER_DATA, and synth.status is the rest?

Reply:
This is fixed up now